frontend/packages/metal3-plugin/src/selectors/baremetal-hosts.ts

```diff
@@ -0,0 +1,42 @@
+import * as _ from 'lodash';
+import { getName } from '@console/shared/src/selectors';
+import { K8sResourceKind, MachineKind } from '@console/internal/module/k8s';
+import { BareMetalHostDisk } from '../types';
+import {
+  HOST_POWER_STATUS_POWERED_ON,
+  HOST_POWER_STATUS_POWERING_OFF,
+  HOST_POWER_STATUS_POWERING_ON,
+  HOST_POWER_STATUS_POWERED_OFF,
+} from '../constants';
+
+export const getHostOperationalStatus = (host: K8sResourceKind) =>
```

Comment:
can you also add types for all of these selectors ?

Reply:
Done